app/views/components/_govspeak-editor.html.erb

```diff
@@ -63,7 +65,7 @@
       describedby: hint_id,
     } %>
   </div>
-  <div class="app-c-govspeak-editor__preview">
+  <div class="app-c-govspeak-editor__preview" aria-live="polite" id="<%= id %>">
```

Comment:
I think that this `id` needs to be different to the textarea `id`

Reply:
@dnkrj updated